src/drivers/DocxViewer.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState, useRef } from 'react';
+import mammoth from 'mammoth';
+import PropTypes from 'prop-types';
+import './DocxViewer.scss';
+import { RESPONSE_REPRESENTATIONS, useFetch } from '../components';
+
+function DocxViewer({ filePath }) {
+  const docxRef = useRef(null);
+  const [error, setError] = useState(null);
+  const { result, isLoading } = useFetch({
+    filePath,
+    responseRepresentation: RESPONSE_REPRESENTATIONS.ARRAY_BUFFER,
+  });
+
+  if (error) {
+    throw error;
+  }
+
+  useEffect(() => {
+    if (isLoading || !result || !docxRef?.current) {
+      return;
+    }
+
+    (async () => {
+      try {
+        const mammothResult = await mammoth.convertToHtml(
```

Comment:
Is this safe ? What happens if I have a <script>maliciousScript()</script> in my docx ?

Reply:
It would just parse it to normal text. The package does not produce script elements.
I don't see any opened security issues in the package.